core/proto/proto.cc

```diff
@@ -413,7 +402,6 @@ com::stripe::rubytyper::FileTable Proto::filesToProto(const GlobalState &gs,
         entry->set_sigil(strictToProto(file.data(gs).originalSigil));
         entry->set_strict(strictToProto(file.data(gs).strictLevel));
         entry->set_min_error_level(strictToProto(file.data(gs).minErrorLevel()));
-        entry->set_compiled(compiledToProto(file.data(gs).compiledLevel));
```

Comment:
IMO we should keep this line, and set it to None unconditionally. This is the default, but it's nice to be explicit.